scrap_zenodo.py

```diff
@@ -5,16 +5,58 @@
 import dotenv
 import pandas as pd
 import argparse
+import yaml
+
 
 def get_arg():
-    """Argument parser
-    Returns:
-        arguments
+    """Argument parser.
+
+    This function parses the name of the input file.
+
+    Returns
+    ----------
+    str
+        Name of the input file.
     """
     parser = argparse.ArgumentParser()
-    parser.add_argument('filetype', nargs='+', metavar='filetype', type=str, help="Filetype(s) of the ZENODO query.")
+    parser.add_argument('input_file', metavar='input_file', type=str, help="Input file.")
     return parser.parse_args()
 
+
+def read_input_file():
+    """Argument parser.
+
+        This function parses the name of the input file.
+
+        Returns
+        ----------
+        md_keywords : list
+            Keywords related to molecular dynamics.
+        generic_keywords : list
+            Generic keywords for zip archives
+        file_types : dict
+            Dictionary with type, engine and keywords to use
+        """
+    arg = get_arg()
+    with open(arg.input_file, "r") as f:
+        data_loaded = yaml.safe_load(f)
+    md_keywords = data_loaded["md_keywords"]
+    generic_keywords = data_loaded["generic_keywords"]
+    file_types = data_loaded["file_types"]
+    return md_keywords, generic_keywords, file_types
+
+
+STRING_MD_KEYWORDS = "AND (" + md_keywords[0]
+for i in range(1, len(md_keywords)):
+    STRING_MD_KEYWORDS += " OR " + md_keywords[i]
+STRING_MD_KEYWORDS += ")"
```

Comment:
Could you check if something like this would work?
```
MD_KEYWORDS = "AND (" + " OR ".join(md_keywords) + ")"
```

Reply:
Many thanks @lisabdj 
Out of curiosity, how many datasets and files do you get with this yaml file?

Reply:
With this yaml file, without the zip query, I get 1126 datasets. Indeed, with the zip query, I get the following message: {'status': 400, 'message': 'Maximum number of 10000 results have been reached.'}. The query stops after 10 000 datasets found.

I counted the number of datasets and files found by filetype, as well as the total datasets (without the zip query and without the duplicates) in the following google sheet : https://docs.google.com/spreadsheets/d/1TTVhbk0SR0wtqR72jSh_JcmcosItA9z1lISXGyIFvBU/edit?usp=sharing